ui/raidboss/data/03-hw/alliance/dun_scaith.ts

```diff
@@ -452,15 +451,14 @@ const triggerSet: TriggerSet<Data> = {
       response: Responses.aoe(),
     },
     {
-      // Ordinarily we wouldn't use a game log line for this.
+      // Ordinarily we wouldn't use a log message for this.
       // However, the RP text seems to be the only indicator.
+      // (Not a MapEffect packet either.)
+      // https://xivapi.com/LogMessage/2747
+      // en: Shadows gather on the floor.
       id: 'Dun Scaith Shadow Links',
-      type: 'GameLog',
-      netRegex: {
-        line: 'Shadows gather on the floor.*?',
-        code: Util.gameLogCodes.message,
-        capture: false,
-      },
+      type: 'SystemLogMessage',
+      netRegex: { id: 'ABB', capture: false },
```

Comment:
Completely unrelated to this PR but if we didn't want to use a `SystemLogMessage` here the only other option is the floor animation which is done via `ActorControl`:
```
41|2024-03-13T14:21:40.4020000-04:00|8003755F|ABB|00|00|00|759f799c80f38925
4096|2024-03-13T14:21:40.4020000-04:00|ActorControl|40000C88|EObjAnimation|00000004|00000008|00000000|00000000|00000000|ca4311e0e521125d
```
```cs
EObjAnimation = 413,
```

Reply:
Makes sense.  How "spammy" of a category is `EObjAnimation`?  Seems like another potential candidate for adding to OP.

Reply:
https://github.com/OverlayPlugin/OverlayPlugin/issues/276#issuecomment-1826398192

Reply:
> [OverlayPlugin/OverlayPlugin#276 (comment)](https://github.com/OverlayPlugin/OverlayPlugin/issues/276#issuecomment-1826398192)

Guessing based on volume that the time period for those totals encompasses multiple days/log files (sorry if that's mentioned somewhere in the comment thread and I missed it).  But 795 for `EObjAnimation` doesn't seem super heavy.

Reply:
Yeah, that was over like a month's worth of logs, but it was also mainly level 90 roulette content or UWU/TEA IIRC.